airbyte-integrations/connectors/destination-motherduck/destination_motherduck/spec.json

```
@@ -0,0 +1,34 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/motherduck",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Destination Duckdb",
+    "type": "object",
+    "required": ["destination_path"],
+    "additionalProperties": true,
+    "properties": {
+      "motherduck_api_key": {
+        "title": "MotherDuck API Key",
+        "type": "string",
+        "description": "API key to use for authentication to a MotherDuck database.",
+        "airbyte_secret": true
+      },
+      "destination_path": {
+        "title": "Destination DB",
+        "type": "string",
+        "description": "Path to the .duckdb file, or the text 'md:' to connect to MotherDuck. The file will be placed inside that local mount. For more information check out our <a href=\"https://docs.airbyte.io/integrations/destinations/duckdb\">docs</a>",
+        "examples": ["/local/destination.duckdb", "md:", "motherduck:"]
+      },
+      "schema": {
+        "title": "Destination Schema",
+        "type": "string",
+        "description": "Database schema name, default for duckdb is 'main'.",
+        "example": "main"
+      }
+    }
+  },
+  "supportsIncremental": true,
+  "supportsNormalization": true,
```

```suggestion
  "supportsNormalization": false,
```

Normalization refers to a post-processing which is now deprecated. Changing to 'false' to reflect this.